src/material-moment-adapter/adapter/moment-date-formats.ts

```diff
@@ -7,16 +7,34 @@
  */
 
 import {MatDateFormats} from '@angular/material';
+import {CdkDateFormats} from '@angular/cdk/datetime';
 
 
 export const MAT_MOMENT_DATE_FORMATS: MatDateFormats = {
+  /**
+   * @deprecated Remove `parse` and use `CDK_MOMENT_DATE_FORMATS` instead.
+   * @deletion-target 8.0.0
+   */
   parse: {
     dateInput: 'l',
   },
   display: {
+    /**
+     * @deprecated Remove `parse` and use `CDK_MOMENT_DATE_FORMATS` instead.
+     * @deletion-target 8.0.0
+     */
     dateInput: 'l',
     monthYearLabel: 'MMM YYYY',
     dateA11yLabel: 'LL',
     monthYearA11yLabel: 'MMMM YYYY',
   },
 };
+
+export const CDK_MOMENT_DATE_FORMATS: CdkDateFormats = {
```

Comment:
you need to provide this in the moment module as well

Reply:
Fixed changes